modules/042-kube-dns/docs/FAQ.md

```diff
@@ -14,4 +14,8 @@ Add the new domain and save the old one:
     - the old clusterDomain.
     - the new clusterDomain.
 1. Wait until kube-apiserver is restarted.
-1. Replace the old clusterDomain with the new one in `dhctl config edit cluster-configuration`
+1. Replace the old `clusterDomain` with the new one in `dhctl config edit cluster-configuration`
+
+**Impotant!** If your Kubernetes version is 1.20 and higher, your controllers in the cluster use [advanced ServiceAccount tokens](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#service-account-token-volume-projection) to work with apiserver. Those tokens have extra fields `iss:` and `aud:` that contain `clusterDomain` (e.g. `"iss": "https://kubernetes.default.svc.cluster.local"`). After changing `clusterDomain` apiserver starts to deny queries with old tokens and controllers are bond to provide errors (including deckhouse). The solution is to wait until Kubernetes rotates the tokens (it will be quite fast despite the expiration date) or restart all pods with controllers.
```

Comment:
The header is "How do I replace the cluster domain without downtime?". But for now, it seems like users will experience downtime or degraded performance. What do you think about changing the header?

Reply:
I agree. Changed.